source/includes/_cvt_events.md

```diff
@@ -0,0 +1,177 @@
+```
+
+> Response Description
+
+```javascript
+{
+  "report_type": "cvt_events",                    // Report Type
+  "report_date": "2020-02-26",                    // Date Report was Run
+  "earliest_date_available": "2018-03-03",        // Earliest Date Data is Available
+  "start_date": "2019-10-01",                     // Start Date of Report
+  "end_date": "2020-02-25",                       // End Date of Report
+  "time_zone": "America/Los_Angeles",             // Time Zone of Report    
+  "interval_size": "calendar_week",               // Interval Size Report is Broken Into
+  "currency": "USD",                              // Currency of Report
+  "report_data": {
+    "campaigns": [
+      {
+        "name": "Ad_Campaign_1",                  // Campaign name
+        "global_master_campaign_id": "USA_1",     // Identifier for campaign
+        "start_date": "2016-07-10",               // Start date for campaign
+        "end_date": "2016-10-24",                 // End date for campaign
+        "type": "Search",                         // Type of campaign
+        "status": "running",                      // Current campaign status
+        "organization": "reachlocal",             // Organization
+        "web_events": [
+          {
+            "web_event_name": "*Book my appointment button clicked",  // Name of web event
+            "web_event_url": "https://www.bookanappointment.com",     // Web event url
+            "cvt_type": "Request",
+            "entry_type": 7,
+            "intervals": [
+                {
+                    "date": "2019-09-30",
+                    "non_qualified_web_events": 4
+                },
+                {
+                    "date": "2019-10-07",
+                    "non_qualified_web_events": 4
+                },
+                {
+                    "date": "2019-10-14",
+                    "non_qualified_web_events": 1
+                }
+            ]
+          },
+          {
+            "web_event_name": "*Request a callback form submitted",
+            "web_event_url": "https://www.bookanappointment.com/callme",
+            "cvt_type": "Submit",
+            "entry_type": 8,
+            "intervals": [
+                {
+                    "date": "2019-09-30",
+                    "qualified_web_events": 1,
+                    "high_web_events": 1
+                },
+                {
+                    "date": "2019-10-07",
+                    "qualified_web_events": 1,
+                    "high_web_events": 1
+                },
+                {
+                    "date": "2019-10-14",
+                    "qualified_web_events": 1,
+                    "high_web_events": 1
+                }
+            ]
+          },
+          {
+            "web_event_name": "*New VDP Pages",
+            "web_event_url": "https://www.vistaford.com/searchnew.aspx",
```

Comment:
Change to a fake URL, this is probably a real customer

Reply:
Changed it